packs/equipment-effects/effect-ancestral-geometry.json

```diff
@@ -0,0 +1,55 @@
+{
+    "_id": "oI7DVO9uKNG4zGtZ",
+    "img": "systems/pf2e/icons/equipment/tattoos/ancestral-geometry.webp",
+    "name": "Effect: Ancestral Geometry",
+    "system": {
+        "badge": {
+            "evaluate": true,
+            "reevaluate": null,
+            "type": "formula",
+            "value": "2d20kh1"
+        },
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Ancestral Geometry]</p>\n<p>Roll 2d20 and record the highest result. Then roll 1d6 and note a type of saving throw: 1–2 Fortitude, 3–4 Reflex, and 5–6 Will.</p>"
```

Comment:
There's a really tortured way to do this in one effect based on the fact that `10 * 1d20 + 1d3` technically encodes all the information in a separable way. If I actually merged it I think stwlam might drive up to Ohio to hit me.

Reply:
Badge: `(10 * 2d20kh1) + 1d3`

For the roll sub: `floor(badge/10)`

For the save type: `ternary(eq(roll remainder,3), "will", ternary(eq(roll remainder, 2), "reflex", "fortitude"))`
where `roll remainder` is `badge - 10 * floor(badge/10)`

Reply:
Maybe you can't. Let's say that's the reason we aren't implementing that terrible idea :D